src/CommonDBTM.php

```diff
@@ -6710,4 +6710,27 @@ public static function displayList(
             'as_map'             => false,
         ]);
     }
+
+    /** @return static[] */
+    public static function getItemsByCriteria(
```

Comment:
`getSeveralFromDBByCrit` may be a better name to match closer with the existing methods.